src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,13 +2,26 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
 
 public class DatabaseManager {
     private final String jdbcDriverClass;
     private final String jdbcUrl;
+    private String schemaPattern = "public";
     private Connection connection;
 
+    public String getSchemaPattern() {
+        return schemaPattern;
```

Comment:
I think name `schemaName` is better.
Maybe schema name should be paramert for gettables(),
What do you think ?